packages/sdk/src/deserialization.ts

```diff
@@ -69,3 +77,271 @@ export function deserializeAccountTransactionPayload(value: Cursor): {
     const payload = accountTransactionHandler.deserialize(value);
     return { type, payload };
 }
+
+export function deserializeCredentialDeploymentValues(
+    serializedPayload: Cursor,
+    data: Partial<UpdateCredentialsPayload>,
+    currentLocation: number
+): Partial<CredentialDeploymentInfo> {    
```

Comment:
Then it's no longer `Partial`

```suggestion
): CredentialDeploymentInfo {    
```